cmd/healthcheck-handler.go

```diff
@@ -62,6 +62,7 @@ func LivenessCheckHandler(w http.ResponseWriter, r *http.Request) {
 		if s.Backend.Type == Unknown {
 			// ListBuckets to confirm gateway backend is up
 			if _, err := objLayer.ListBuckets(ctx); err != nil {
+				logger.LogIf(ctx, err)
```

Comment:
We specifically removed this because when the backend is disconnected or down, this floods the user's console.

Reply:
> We specifically removed this because when the backend is disconnected or down, this floods the user's console.

I guess we can use log once feature here